src/libbacktrace/Makefile.am

```diff
@@ -6,12 +6,12 @@
 # met:
 
 #     (1) Redistributions of source code must retain the above copyright
-#     notice, this list of conditions and the following disclaimer. 
```

Comment:
libbracktrace is not part of Rust proper, so the Rust styleguide doesn't apply (and these files shouldn't be modified anyway if at all possible, because it complicates upgrades).